src/lib/helpers.ts

```diff
@@ -87,9 +87,13 @@ export function decryptStrHash(password, encryptedString) {
 		return false;
 	}
 }
-
-export function buf2hex(buffer) {
-	return Array.prototype.map.call(new Uint8Array(buffer), (x) => ("00" + x.toString(16)).slice(-2)).join("");
+ 
+export function buf2hex(buffer : ArrayBuffer) {
```

Comment:
The issue for the `any` types was due to `buf2hex`  not written properly in typescript. In the near future all functions in `helper.ts`  should be rewritten in proper typescript